packages/core/src/constants/type/Config.ts

```diff
@@ -12,7 +12,7 @@ export type FullConfig = {
   readOnlyChainId?: ChainId
   readOnlyUrls?: NodeUrls
   multicallAddresses?: MulticallAddresses
-  supportedChains: number[]
+  networks: Chain[]
```

Comment:
For backward compatibility I would do the following thing:
1) keep both `supportedChains` and introduce `networks`
2) Add deprecation warning to usage of `supportedChains`
3) Add custom logic to useConfig, that will populate networks if empty based `supportedChains`